content/200-orm/200-prisma-client/300-client-extensions/100-model.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: '`model`: Add custom methods to your models'
+title: 'Add custom methods to Prisma models'
```

Comment:
Don't these changes take away from the meaning of the title, as they previously better expressed what kind of client extension they're about?